docker-compose.yaml

```
@@ -0,0 +1,94 @@
+version: "3.6"
+services:
+  # CB-Spider
+  cb-spider:
+    image: cloudbaristaorg/cb-spider:0.9.0
+    container_name: cb-spider
+    platform: linux/amd64
+    ports:
+      - "1024:1024"
```

```suggestion
      - "${SP_HOST_PORT}:1024"
```
컨테이너 컴포넌트의 호스트 포트를 환경변수로 제공하면 좋을 것 같습니다.